src/envoy/http/path_matcher/filter.cc

```diff
@@ -49,8 +49,13 @@ Envoy::Http::FilterHeadersStatus Filter::decodeHeaders(
     return Envoy::Http::FilterHeadersStatus::StopIteration;
   }
 
-  std::string method(utils::getRequestHTTPMethodWithOverride(
-      headers.Method()->value().getStringView(), headers));
+  bool usedMethodOverride = utils::handleHttpMethodOverride(headers);
+  if (usedMethodOverride) {
```

Comment:
Just. if (utils::handleHttpMethodOverride(headers)) and add a comment

Reply:
Done